### PR TITLE
Add addFileFromString method.

### PR DESCRIPTION
**Pull request by Pato05**
Just a little function to add a file with a custom fileName and content-type

### CODE REVIEW

File: src/Body/FormBody.php [hunk 1]
```diff
@@ -150,8 +165,7 @@ private function getMultipartFieldArray(): array
     private function generateMultipartFileHeader(string $name, string $fileName, string $contentType): string
     {
         $header = "Content-Disposition: form-data; name=\"{$name}\"; filename=\"{$fileName}\"\r\n";
-        $header .= "Content-Type: {$contentType}\r\n";
-        $header .= "Content-Transfer-Encoding: binary\r\n\r\n";
```
Comment (kelunik) [inline src/Body/FormBody.php:154]: Why did you remove this line?

File: src/Body/FormBody.php [hunk 2]
```diff
@@ -164,6 +178,7 @@ private function generateMultipartFieldHeader(string $name, string $contentType)
         } else {
             $header .= "\r\n";
         }
+        $header .= "\r\n";
```
Comment (kelunik) [inline src/Body/FormBody.php:181]: Why did you add this line?

**Comment by kelunik** [comment]
Merged manually, thanks!

File: src/Body/FormBody.php (hunk 1 above)
Comment (Pato05) [reply src/Body/FormBody.php:154]: Because afaik Content-Transfer-Encoding is deprecated and about Content-Type I think I readded it?

File: src/Body/FormBody.php (hunk 2 above)
Comment (Pato05) [reply src/Body/FormBody.php:181]: Just to make the code look nicer, I removed the carrier return on the upper parts

**Comment by Pato05** [comment]
Ok, thank you :D

File: src/Body/FormBody.php (hunk 1 above)
Comment (kelunik) [reply src/Body/FormBody.php:154]: Yeah, not sure whether this is still needed for anyone, but it doesn't really hurt, either.

File: src/Body/FormBody.php (hunk 2 above)
Comment (Pato05) [reply src/Body/FormBody.php:181]: Oh yea, sorry, my fault xd

File: src/Body/FormBody.php (hunk 1 above)
Comment (Pato05) [reply src/Body/FormBody.php:154]: Yeah, true... Indeed I kinda removed it to do some tests, then I saw that it was deprecated so I didn't rebring it there.
